model/src/pyrenew/deterministic/process.py

```diff
@@ -29,14 +30,28 @@ def sample(
         Returns
         -------
         tuple
-            Containing the stored values during construction.
+            Containing the stored values during construction wrapped in a SampledValue.
```

Comment:
This doesn't read well to me. Not sure what it is trying to say.

Reply:
Made an attempt in 9ff9dea but I increasingly think we should get rid of the `DeterministicProcess` class, as I think it should be replaced by other more time-aware ways of handling temporal broadcasting of values. Beyond the scope of this PR, though.